<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translatorwindow</class>
 <widget class="QWidget" name="translatorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>我的翻译任务</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tMysectiontask">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>241</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="tComment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>241</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>负责人的评语：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tMywork">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>241</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>我的最新翻译成果：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bSubmit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bReturn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
